uberfire-io/src/main/java/org/uberfire/io/IOService.java

```diff
@@ -84,6 +84,19 @@ FileSystem newFileSystem(final URI uri,
             throws IllegalArgumentException, FileSystemAlreadyExistsException,
             ProviderNotFoundException, IOException, SecurityException;
 
+    default FileSystem getOrNewFileSystem(final URI uri, final Map<String, ?> env)
```

Comment:
Would you please add a test for this method?

Reply:
Sure I can!
I forgot to test this new method, thanks for pointing it out!

Reply:
Done